shared/common/src/androidUnitTest/kotlin/cz/quanti/common/di/CheckModulesTest.kt

```diff
@@ -0,0 +1,21 @@
+package cz.quanti.common.di
+
+import org.koin.core.annotation.KoinExperimentalAPI
+import org.koin.test.verify.verify
+import kotlin.test.Test
+
+@OptIn(KoinExperimentalAPI::class)
+class CheckModulesTest {
```

Comment:
podle me dava smysl mit jen jeden CheckModulesTest a to na urovni app

Reply:
Udelal jsem rfc, dal jsem ho zvlast do PR https://github.com/alesrazym/rocket-app/pull/21. Vice v komentech tam.